lib/src/components/Avatar/avatar.module.scss

```diff
@@ -0,0 +1,76 @@
+@layer components {
+  .root {
+    flex-shrink: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+    overflow: hidden;
+    aspect-ratio: 1 / 1;
+    border-radius: 50%;
+
+    width: var(--avatar-width);
+    height: var(--avatar-width);
+    font-size: calc(var(--avatar-width) / 2.5);
+
+    img {
+      object-fit: cover;
+      width: 100%;
+      height: 100%;
+    }
+
+    p {
```

Comment:
```suggestion
    span or maybe the text class ? 
```

Reply:
the Text component class is `wui-text`. We could definitely rely on it